src/Data/Textile/Knitting.elm

```diff
@@ -94,10 +94,10 @@ toLabel knittingProcess =
             "Tricotage moyen (par défaut)"
 
         FullyFashioned ->
-            "Fully fashioned"
+            "Fully fashioned / Seamless"
 
-        Seamless ->
-            "Seamless"
+        Integral ->
+            "Intégral / Whole garment"
```

Comment:
Do we keep the English terminology because *Whole garment* isn't directly translatable to eg. `vêtement complet`?

Reply:
I believe it's the technical term used, the same way as `Seamless` (sans coutures) was ?